recipes/spleeter-gpu/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "spleeter-gpu" %}
+{% set version = "1.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 49e776f26c561357eb9608a459030ab46cfe11edf21fb17470630a0d3777d453
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [osx or py<36 or py>37]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - requests
+    - setuptools
+    - importlib_resources  # [py<37]
+    - tensorflow-gpu ==1.14.0
```

Comment:
Does this need to be GPU only and pinned to this exact version?

Reply:
Yes it does, this recipe is a GPU specific version that rely on tensorflow-gpu. As for spleeter original recipe the Tensorflow version is frozen as inner ML model rely on this specific version at the moment